tests/vmcheck/test.sh

```diff
@@ -56,7 +56,7 @@ vm_cmd ostree refs vmcheck_tmp vmcheck_remote --delete
 # comes with the distro to help speed up rpm-ostree metadata fetching since we
 # don't cache it (e.g. on Fedora, it takes *forever* to fetch metadata, which we
 # have to do dozens of times throughout the suite)
-if ! vm_cmd test -f /etc/yum.repos.d/.vmcheck; then
+if vm_cmd test -d /etc/yum.repos.d/ && ! vm_cmd test -f /etc/yum.repos.d/.vmcheck; then
```

Comment:
Seems simpler to just do `mkdir -p /etc/yum.repos.d` before all this logic?

Do we care about removing it afterwards?

Reply:
Damn, 15 seconds.